spec/spec_helper.rb

```diff
@@ -177,4 +177,21 @@ def main_app
   config.include FactoryGirl::Syntax::Methods
 
   config.infer_spec_type_from_file_location!
+
+  config.expect_with :rspec do |expectations|
+    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
+  end
+
+  config.default_formatter = 'doc' if config.files_to_run.one?
+
+  config.order = :random
+  Kernel.srand config.seed
```

Comment:
This'll be interesting... I don't think we were doing random before, but it's a good idea.